src/app/(public)/register-completion/page.tsx

```diff
@@ -0,0 +1,306 @@
+'use client'
+
+import { Button } from '@/components/ui/button'
+import {
+  Card,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from '@/components/ui/card'
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/form'
+import { Input } from '@/components/ui/input'
+import { LoadingSpinner } from '@/components/ui/loadingSpinner'
+import { PhoneInput } from '@/components/ui/phoneInput'
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '@/components/ui/select'
+import { useToast } from '@/components/ui/use-toast'
+import { cityList } from '@/constants/trCity'
+import {
+  getAdminRegistrationApplicationSummary,
+  postRegistrationApplication,
+} from '@/modules/adminRegistrationApplications/service'
+import { zodResolver } from '@hookform/resolvers/zod'
+import { EyeIcon, EyeOffIcon } from 'lucide-react'
+import Image from 'next/image'
+import { useRouter } from 'next/navigation'
+import { useEffect, useState } from 'react'
+import { useForm } from 'react-hook-form'
+import { useTranslation } from 'react-i18next'
+import { z } from 'zod'
+
+interface Iprops {
+  id: string
+}
+
+const RegisterCompletion: React.FC<Iprops> = ({ id }) => {
+  const { t } = useTranslation()
+  const { toast } = useToast()
+  const [loading, setLoading] = useState(false)
+  const [instName, setInstName] = useState<string>('')
+  const [showPassword, setShowPassword] = useState(false)
+  const router = useRouter()
+
+  const PhoneNumberSchema = z.object({
+    countryCode: z.string().min(1),
+    lineNumber: z.string().min(1),
+  })
+
+  const InstitutionFormSchema = z.object({
+    firstName: z
+      .string()
+      .min(1, t('requiredField'))
+      .min(3, t('minLength', { field: 3 }))
+      .max(255, t('maxLength', { field: 255 })),
+    lastName: z
+      .string()
+      .min(1, t('requiredField'))
+      .min(3, t('minLength', { field: 3 }))
+      .max(255, t('maxLength', { field: 255 })),
+    emailAddress: z
+      .string()
+      .min(1, t('requiredField'))
+      .email(t('invalidEmail')),
+    city: z.string().min(1, t('requiredField')),
+    password: z
+      .string()
+      .min(1, t('requiredField'))
+      .min(8, t('minLength', { field: 8 }))
+      .max(50, t('maxLength', { field: 50 })),
+    phoneNumber: PhoneNumberSchema,
+  })
```

Comment:
formSchema için ayrı bir dosya oluşturup oradan çağıralım. Örneği şuan mevcut main branch'ından incelenebilir.